tests/unit/helpers.js

```diff
@@ -167,6 +167,8 @@ export const userAgentsByPlatform = {
       rv:33.0) Gecko/20100101 Firefox/33.0`,
     firefox57: oneLine`Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:57.0)
       Gecko/20100101 Firefox/57.1`,
+    firefox61: oneLine`Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:57.0)
```

Comment:
It would be nice to change this to `rv:61.0` just to avoid confusion. I see the line above it needs changing too 🤒

Reply:
heh, oops